### DISCUSSION TITLE
Running Django-helpdesk using Docker containers

### DISCUSSION THREAD

**Discussion by trainor1** [django-helpdesk/django-helpdesk#1105] [Q&A]
Hi All,

I am getting ready to roll out Django-helpdesk on my new VPS.  I am planning to use Docker containers.  My experience with Docker containers is low.  So, I am interested in any advice, or pointers to resources that would help me.  I am particularly interested in the best approach to running manage.py get_email in a containerized setup.

Thanks for your help.

Best,

Kevin

**Comment by trainor1** [comment]
Hi All,

Do you know of anyone who has deployed Django-helpdesk using Docker containers?  Can you point to resources on this?

Best,

Kevin

**Reply by brucegibbins** [reply]
I'm afraid I haven't any first-hand knowledge.

**Comment by Benbb96** [comment]
Hi @trainor1,

I tried to do it once in order to deploy a demo on render.com

You can take a look at [this branch](https://github.com/Benbb96/django-helpdesk/tree/dockerize/demo) on my fork (see the Dockerfile and settings.py). I may help you but I never managed to successfully run a complete  version of djando-helpdesk :smile:

**Comment by uhurusurfa** [comment]
There is now a Dockerfile in the project and hopefully that answers this issue so closing
